app/assets/stylesheets/index.scss

```diff
@@ -1,16 +1,31 @@
- .title {
+@import "colors";
+@import "foundation_and_overrides";
+
+.title {
   font-size: 20px;
   margin-bottom: 0;
 }
 
 .piece {
-  padding-left: 10px;
-  padding-top: 20px;
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border-radius, color, font-weight, margin-bottom, margin-left, margin-right, padding-bottom, padding-bottom, padding-left, padding-top